hvplot/ui.py

```diff
@@ -551,9 +560,22 @@ def _plot(self, *events):
         finally:
             self._layout.loading = False
 
+    def _code(self):
+        self.code = self._build_code_snippet()
+        self._code_pane.object = f"""```python\nimport hvplot.{self._backend}\n\n{self.code}\n```"""
```

Comment:
I would not include the import, it doesn't bring much in my opinion.

Reply:
I disagree; it reduces the friction for the user to immediately get started--else, the user needs to manually type `import hvplot.pandas`

If they do not want the import, they can simply copy up to the line below.

Reply:
IIRC you have to run the import first in a notebook for the explorer to work. What use case do you have in mind?

Reply:
This doesn't necessarily have to run in notebook; they can copy it to a script or in the future, if they launch the explorer from CLI.